ontology/yaml/resources/HVAC/entity_types/FCU.yaml

```diff
@@ -1544,7 +1544,17 @@ FCU_RTM_DXZTC_SRC:
   uses:
   - filter_alarm
   - lost_power_alarm
-
+  
+FCU_DXZTC:
+  description: "FCU with SS, discharge temp sensors, zone control and sensor, filter change"
+  is_canonical: true
+  implements:
+  - FCU
```

Comment:
Thanks for pointing this out sir, I actually need schedule_run_command and DFSS has it. Will add this now sir, but I don't actually know why the whole DFSS tag is needed instead of just adding schedule_run_command. My points list are:

![FCU](https://user-images.githubusercontent.com/21256796/224628635-63c7b82b-d481-46a6-acfb-8d1aba5f36d5.png)

I may have missed reading it in documentation on DBO or docs might have been updated that I didn't know, sorry about that.